kbs/src/http/attest.rs

```diff
@@ -22,6 +41,14 @@ pub(crate) async fn auth(
 ) -> Result<HttpResponse> {
     info!("Auth API called.");
     debug!("Auth Request: {:?}", &request);
+    let version = Version::parse(&request.version).unwrap();
+    if !VERSION_REQ.matches(&version) {
+        raise_error!(Error::ProtocolVersion(format!(
+            "client {}{} server",
```

Comment:
nit: is this how the string is supposed to look?

Reply:
It becomes like this:

```
ErrorInformation { error_type: \"https://github.com/confidential-containers/kbs/errors/ProtocolVersion\", detail: \"KBS Client Protocol Version Mismatch: client 0.1.1<=0.1.0 server\" }")
```

I iterated over a few alternatives. I'm open to any suggestions :-)

Reply:
pushed an update.